lib/Backend/GlobOptFields.cpp

```diff
@@ -3220,6 +3220,12 @@ GlobOpt::UpdateObjPtrValueType(IR::Opnd * opnd, IR::Instr * instr)
         return;
     }
 
+    ValueInfo *objValueInfo = objVal->GetValueInfo();
```

Comment:
if something is a bytecode constant, shouldn't the valueType be definite (and the check above pass)?

Reply:
As discussed offline, added comment below.